src/approx_itensornetwork/density_matrix.jl

```diff
@@ -335,6 +328,97 @@ function _rem_vertex!(
   return U
 end
 
+function _update_cache_w_low_rank_projector!(
+  ::Algorithm"direct_eigen",
+  alg_graph::_DensityMartrixAlgGraph,
```

Comment:
Just noticed this should be `_DensityMatrixAlgGraph`, not `_DensityMartrixAlgGraph`. Also I think it's best to remove the underscore.